src/types.ts

```diff
@@ -140,6 +140,27 @@ export interface LoadConfigOptions<
     | {
         extendKey?: string | string[];
       };
+  /**
+   * Options for extending configs
+   */
+  extendOptions?: {
+    /**
+     * Skip and ignore remote configs when extending. This blocks downloading configs
+     * from github or npm, only local configs will be used.
+     */
+    skipRemote?:
```

Comment:
Since at least for now use `giget` for giget options, maybe we can simply support `giget: false` (i like to experiment new namespace but it seems little bit overwork and npm less relavant to _remote_  term...

Reply:
Thanks for the feedback. I added only `giget: false`, let me know if there is something else